host_core/.credo.exs

```diff
@@ -0,0 +1,211 @@
+# This file contains the configuration for Credo and you are probably reading
```

Comment:
Is this different from the default config?

Reply:
Yes there are a number of checks that I've made more strict than the defaults.

Reply:
note that not all of the credo recommendations have been implemented. In the interest of stabilization, I'll come back to them individually in future PRs.